cluster/gce/cloud-init/master.yaml

```diff
@@ -45,6 +45,9 @@ write_files:
       # in the kernel. We recommend using cgroups to do container-local accounting.
       LimitNPROC=infinity
       LimitCORE=infinity
+      # Comment TasksMax if your systemd version does not supports it.
+      # Only systemd 226 and above support this version.
```

Comment:
I think the comment may be redundant; we originally added the comment because the service should fail to start if it was set, but from testing, it only printed a _warning_, not an error; https://github.com/moby/moby/pull/24100#issuecomment-229669421

Current versions of our unit file set this option unconditionally, and (testing on CentOS 7.6, which has systemd 219), work just fine.

Reply:
Done